pipeline/translate/splitter.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+"""
+Splits a dataset to chunks. Generates files in format file.00.zst, file.01.zst etc.
+
+Example:
+    python splitter.py \
+        --output_dir=test_data \
+        --compression_cmd=zstd \
+        --num_parts=10 \
+        --output_suffix=.ref \
+        test_data/corpus.en.zst
+"""
+
+import argparse
+import os
+import subprocess
+from contextlib import ExitStack
+
+
+def split_file(mono_path, output_dir, num_parts, compression_cmd, output_suffix=""):
+    os.makedirs(output_dir, exist_ok=True)
```

Comment:
Suggestion: Honestly my preference here would be to throw an error here if the path doesn't exist. I would prefer to be stricter with our input validation, rather than permissive.

Reply:
A pipeline script usually supposed to create am output directory if it doesn't exist. This might change on Taskcluster because we usually put output to "artefacts" folders that already exists. I'd still keep this behavior for now.